src/components/SignupForm/useSignupFormValidation.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import SignUpFormDataInterface from './interfaces/SignupFormDataInterface';
+import ErrorInterface from './interfaces/ErrorInterface';
+import ValidationInterface from './interfaces/ValidationInterface';
+
+const useSignupFormValidation = (data: SignUpFormDataInterface): ValidationInterface => {
+  const initialErrorData: ErrorInterface = {
+    isError: false,
+    errorText: ''
+  };
+
+  const [error, setError] = useState(initialErrorData);
+
+  const validateForm = (): boolean => {
+    const { name, email, password, repeatPassword } = data;
+
+    if (name.trim() === '' || email.trim() === '' || password.trim() === '' || repeatPassword.trim() === '') {
```

Comment:
name.trim() === '' || email.trim() === '' || password.trim() === '' || repeatPassword.trim() === '' - it could be a function `isAnyEmpty`

Reply:
it could be also const